### ISSUE TITLE
docstring has wrong usage of unescaped single quotes in mmm-auto.el and mmm-vars.el

### ISSUE THREAD

**Issue by vinc17fr** [dgutov/mmm-mode#136]
When started for the first time (or after the `.emacs.d/eln-cache` directory is removed), Emacs 29 outputs warnings during the compilation of the `.el` files:
```
⛔ Warning (comp): mmm-vars.el:869:2: Warning: defvar `mmm-classes-alist' docstring has wrong usage of unescaped single quotes (use \= or different quoting)
⛔ Warning (comp): mmm-auto.el:168:2: Warning: docstring has wrong usage of unescaped single quotes (use \= or different quoting)
```

**Comment by dgutov** [comment]
Hi!

Which versions are you trying? I think this was fixed in 0.5.9: 869d5ef and d5af05a.

**Comment by vinc17fr** [comment]
I'm using Debian's version (unstable). Since the latest changes weren't recent, I thought that Debian was using the latest version, but actually it still uses 0.5.8! Sorry about that.
